src/forms/calendarComponent.story.jsx

```diff
@@ -3,10 +3,13 @@ import CalendarComponent from './CalendarComponent';
 import { storiesOf } from '@storybook/react';
 import { decorateWithBasics, decorateWithInfo } from '../utils/decorators';
 import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { LocalDate, convert } from 'js-joda';
 
 import Section from '../layout/Section';
 import Chunk from '../layout/Chunk';
 
+const jodaDate = LocalDate.now();
```

Comment:
You probably already know, but just wanted to point out that if you do something like this in a unit test that has a snapshot, you'll want to mock any `.now` call in js-joda so that it returns a fixed value so that the snapshot doesn't change each time it's run - we've done that in a few places in mup-web, e.g. https://github.com/meetup/mup-web/blob/master/src/app/group/event/schedule/selectors/selectInitialValues.test.js#L19-L27

Just for future reference